### DISCUSSION TITLE
Espanso doesn<t start for a regular user

### DISCUSSION THREAD

**Discussion by lduperval** [espanso/espanso#2166] [Q&A] [answered]
Hi,

 I installed 2.2.1 for a regular user account on my PC (i.e. not an admin) but when I launch espanso, nothing happens. I look at the Task Manager and I only see the version running for the admin user.

What am i missing?

L

**Comment by smeech** [comment, marked as answer]
I think one can only run one instance of Espanso.

What's your OS?

https://github.com/espanso/espanso/issues/1863 may be relevant.

**Reply by lduperval** [reply]
Windows  11. 

Ah, you are correct! I killed the other espanso daemons and it worked. Well, that's an easy enough workaround.

It would be nice to be able to run multiple instances. Since I'm the sole user of my PC, it's not a problem. But on a shared machine, I can see how it would be highly annoying.

L

**Reply by smeech** [reply]
You're welcome. 

#1863 remains open, so might attract some work, although the developer team is very small, with other priorities in hand. For now the [workaround](https://github.com/espanso/espanso/issues/1863#issuecomment-1973922754) is likely to remain the only option, I'm afraid.
